.github/workflows/release.yml

```diff
@@ -0,0 +1,142 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Tag for release
+        required: true
+
+jobs:
+  build-binary:
+    name: Build Binary
+    runs-on: self-hosted
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Setup Rust
+        run: rustup show
+      - name: Build
+        run: |
+          cargo build --locked --release --features turing-node
+          mkdir -p artifacts
+          cp target/release/oak-collator artifacts/
+      - name: Upload binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: oak-collator
+          path: artifacts
+  build-runtime:
+    name: Build Runtime
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chain: ["turing"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Build
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.4.0
+        with:
+          chain: ${{ matrix.chain }}
+      - name: Summary
+        run: |
+          echo '${{ steps.srtool_build.outputs.json }}' | jq . > ${{ matrix.chain }}-srtool-digest.json
+          cat ${{ matrix.chain }}-srtool-digest.json
+          cp ${{ steps.srtool_build.outputs.wasm_compressed }} ${{ matrix.chain }}-runtime.wasm
+      - name: Upload runtime
+        uses: actions/upload-artifact@v2
+        with:
+          name: turing-runtime
+          path: |
+            ${{ matrix.chain }}-runtime.wasm
+            ${{ matrix.chain }}-srtool-digest.json
+  build-docker:
+    name: Build Docker
+    runs-on: ubuntu-latest
+    needs: build-binary
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.tag }}
+      - name: Download binary
+        uses: actions/download-artifact@v2
+        with:
+          name: oak-collator
+      - name: Build
+        run: |
+          docker build -f ./docker/turing/Dockerfile -t oaknetwork/oak-testchain:${{ github.event.inputs.tag }} .
+          docker tag oaknetwork/oak-testchain:${{ github.event.inputs.tag }} oaknetwork/oak-testchain:latest
+          docker push oaknetwork/oak-testchain:${{ github.event.inputs.tag }}
+          docker push oaknetwork/oak-testchain:latest
```

Comment:
Assuming that `oak-testchain` is a leftover value from testing this in another repo.

Reply:
Yup. We'll update before merge, but I don't want to ship Docker images to a repository I don't have access to delete in.